packages/nimble-components/src/wafer-map/modules/prerendering.ts

```diff
@@ -51,7 +51,29 @@ export class Prerendering {
         const isDieRenderInfo = (
             info: DieRenderInfo | null
         ): info is DieRenderInfo => info !== null;
-        this._diesRenderInfo = this.wafermap.dies
+        if (this.wafermap.diesTable === undefined) {
+            this._diesRenderInfo = this.wafermap.dies
+                .map(die => this.computeDieRenderInfo(die))
+                .filter(isDieRenderInfo);
+            return;
+        }
+        // will chnange prerendering info for the new strategy in the following PR
+        this._diesRenderInfo = (
+            this.wafermap.diesTable.toArray() as {
+                colIndex: number,
+                rowIndex: number,
+                value: string
+            }[]
+        )
+            .map(row => {
+                // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+                return {
+                    x: row.colIndex,
+                    y: row.rowIndex,
+                    value: row.value,
+                    tags: []
+                };
+            })
```

Comment:
this all looks super sketchy. approving assuming it will be heavily cleaned up and will not have unsafe assignments and `as` casting is the following PR. If that is not the case should look more carefully and try and clean-up

Reply:
This is just an adaptor to see that both strategies produce the same results with the previous render. It's not required, I can remove it